app/src/main/res/layout/fragment_homepage.xml

```diff
@@ -58,13 +59,13 @@
             <TextView
                 android:id="@+id/Wolf_info"
                 android:layout_width="243dp"
-                android:layout_height="105dp"
+                android:layout_height="76dp"
                 android:gravity="left"
-                android:text="Any wolf logo effortlessly switches the backdrop, toggling between a crisp white canvas and a sleek black background. (← try me)"
+                android:text="The wolf logo switches the backdrop, toggling between a white background and a black background. "
```

Comment:
here you're using the word background twice in five words, so the flow of words could be improved, maybe pulling on your audiences background knowledge of light and dark more and replace "white background and a black background" with light to dark mode